content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -0,0 +1,319 @@
+---
+title: Applying Pod Security Standards at the cluster level
+content_type: tutorial
+weight: 10
```

Comment:
nit: both articles have the same weight, is that intended?

Reply:
Yes